app/controllers/PasskeyController.scala

```diff
@@ -31,9 +31,14 @@ class PasskeyController(
     case Mode.Prod => "Janus-Prod"
   }
 
+  def showRegistrationPage: Action[AnyContent] = authAction {
+    implicit request =>
+      Ok(views.html.passkeyRegistration(request.user, janusData))
+  }
```

Comment:
It looks as though this action is redundant now as the view isn't used, unless I'm missing something.